ToznyRemoteRealmAPI.php

```diff
@@ -55,9 +55,16 @@ function __construct($realm_key_id, $realm_secret, $in_api_url = NULL)
     {
         $this->setRealm($realm_key_id, $realm_secret);
 
-        if ($in_api_url) {
+        $apiTmp = getenv("INTERNAL_API_URL");
```

Comment:
Ultimately, this is because we use environment variable files for Rancher setup. In this case, the client is the Admin service, which still needs to know what the internal api url is. The best way with our setup was to pass this in via an environment variable to our docker container at runtime.